edifact-edi/2.2/modules/ROOT/pages/index.adoc

```diff
@@ -55,20 +56,19 @@ your schema location is `/mypartner/ORDERS.esl`.
 
 == The Hierarchy of EDIFACT Connector Message Structure
 
-Use this connector to read or write EDIFACT documents into (or from) the canonical EDI message structure. This structure is a hierarchy of Java Maps and Lists. You use either DataWeave or code to manipulate them. Each transaction has its own structure as defined in the schemas.
+Use this connector to read or write EDIFACT documents in (or from) the canonical EDI message structure. This structure is a hierarchy of Java Map and List objects. You use either DataWeave or code to manipulate them. Each transaction has its own structure as defined in the schemas.
 
-The message contains the following keys, some of which apply to either the Read operation or Write operation, as indicated:
+The message contains the following keys, some of which apply to either the Read operation or the Write operation, as indicated:
 
 [%header%autowidth.spread]
 |===
 |Key Name |Description
-|Delimiters |An optional set of delimiters used in the message. If used in a read operation, the delimiters are based on the most recent interchange processed by a read operation. If used for a write operation, the delimiters override the values in the module configuration. The characters in the string of delimiters are interpreted based on their position in string, in the following order:
-data element separator, component element separator, repetition separator, segment terminator, release character.
+|Delimiters |An optional set of delimiters used in the message. If used in a Read operation, the delimiters are based on the most recent interchange processed by a Read operation. If used for a write operation, the delimiters override the values in the module configuration. The characters in the string of delimiters are interpreted based on their position in string, in the following order: data element separator, component element separator, repetition separator, segment terminator, release character.
```

Comment:
Should "write operation" be "Write operation" for consistency with "Read operation"?